index.js

```diff
@@ -520,6 +520,7 @@ class ClientServiceBase {
 
               return resolve(new ContractExecutionResult(
                   ledgerResult.getResult(),
+                  ledgerResult.getFunctionResult(),
                   ledgerResult.getProofs(),
                   auditorResult.getProofs(),
```

Comment:
Not really sure, but I guess we should use `getContractResult()`, `getLedgerProofs()` and `getAuditorProofs()` here since these are deprecated? It might be troublesome to use the `ContractExecutionResult` constructor properly, though.

Reply:
@jnmt 
Thank you for your catch.
I fixed it in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/79/commits/0a34e9ed25632f7eb3d33152792e15442e9a8c31


- The `getProofs()` should be replaced by `getLedgerProofs()`. It's a bit tricky but we need to use `getLedgerProofs()` to get the AssetProofs of the ContractExecutionResult from Auditor here internally.
- `getResult()` should be replaced by `getContractResult()`. Sorry I didn't notice it earlier because the string output and the object output from Node.js look too similar. It was a bug.

Reply:
Sorry for the additional one. It seems that `_validateResponses()` also uses the deprecated interfaces. Can you briefly check if there are similar cases in the same file?
Also, as you notice,  it is a little weird to call `getLedgerProofs()` for the auditor one. I don't know/remember the reason why we don't use the GRPC responses directly to compare the results. Do you know the reason? If it's easy to use them, we might not have to do the tricky thing similar to the Java SDK `validateResponses()`.

Reply:
@jnmt 
Thanks for your confirmation!
To be honest, I can't remember the reason.
I guess we tried to implement the SDK as similar to Java's implementation.

I revise this part with your idea in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/79/commits/af09be37861972232ae6648d1787025b730c0bf0

Just using the gRPC-generated JS object (proto.rpc.ContractExecutionResponse) seems easier to me too.